git-event/handler.go

```diff
@@ -55,6 +56,14 @@ func Handle(req []byte) string {
 
 	}
 
+	auditEvent := sdk.AuditEvent{
+		Message: "Garbage Collection Completed",
+		Owner:   event.Installation.Account.Login,
```

Comment:
If `eventHeader ` is different than `installation_repositories`, then GC will not be completed and the message can be confusing, the empty owner as well.

If the message goes into a variable, that can solve the problem.